seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf

```diff
@@ -33,8 +33,25 @@ env {
 source {
   # This is a example input plugin **only for test and demonstrate the feature input plugin**
   FakeSource {
+    fields {
+      c_map = "map<string, string>"
+      c_array = "array<tinyint>"
+      c_string = string
+      c_boolean = boolean
+      c_tinyint = tinyint
+      c_smallint = smallint
+      c_int = int
+      c_bigint = bigint
+      c_float = float
+      c_double = double
+      c_decimal = "decimal(30, 8)"
+      c_null = "null"
+      c_bytes = bytes
+      c_date = date
+      c_time = time
+      c_timestamp = timestamp
+    }
     result_table_name = "fake"
-    field_name = "name,age,timestamp"
   }
```

Comment:
Should you change all testcase?  

https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/resources/examples
https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples
https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/resources
https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-spark-connector-v2-e2e/src/test/resources

Reply:
yes, I do, but I have not test it, I will test as soon as possible。

Reply:
> Should you change all testcase?
> 
> https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/resources/examples https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/resources https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-spark-connector-v2-e2e/src/test/resources

Thx，I have changed all of them. Is there has  auto-test by CI，the test is so much.